go/linux/13.0/web.config

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we move this file up to the linux folder and use a match on `\d+.\d+` so we don't have to keep doing this for each version? If it diverges then we can make a change at that point...

We should do this for the Windows links as well

Reply:
Done for Linux.

Question re: Windows links:
go/desktop/10.0/web.config has a redirect for issue-1285 that isn't in later versions. Should it be?

Reply:
I went ahead and consolidated the desktop and developer go links 
(except for links to help.keyman.com/developer/13.0 which don't exist yet)